src/breeze/actions/emails/show.cr

```diff
@@ -0,0 +1,38 @@
+abstract class Breeze::Emails::Show < BreezeAction
+  param plain : Bool = false
+
+  get "/emails/:email_name" do
+    email = load_email_template(email_name)
+
+    if plain
+      if text_body = email.text_body
+        plain_text(text_body, status: 200)
+      else
+        render_missing_template_error(email, "text")
+      end
+    else
+      if html_body = email.html_body
+        send_text_response(
+          html_body,
+          content_type: "text/html",
+          status: 200
+        )
+      else
+        render_missing_template_error(email, "html")
+      end
+    end
+  end
+
+  abstract def load_email_template(email_name : String) : Carbon::Email
```

Comment:
You must define this method and make sure it returns the `Carbon::Email` instance. it might look like this:

```crystal
class ShowEmails < Breeze::Emails::Show
  def load_email_template(email_name : String) : Carbon::Email
    user = UserQuery.first
    email = if email_name == "WelcomeEmail"
              WelcomeEmail.new(user)
            else
              PasswordResetRequestEmail.new(user)
            end

    email
  end
end
```

Reply:
Do you think we should namespace this under something like `Breeze::CarbonEmails::Show` or something, to make it more clear that this only supports Carbon ATM?

Or are you thinking we could support other adapters with some simple method overloading in the future and still just keep the idea of a Breeze email contained in a single class?

Reply:
We would only support Carbon. If you wanted to use a different email adapter, then it would probably be up to you to figure out how to get it in there. But I do have plans for a way to extend Breeze, so that could be an alternative extension 🤔